### PR TITLE
Fix: PatientNoteCard Text Overflowing and Current Address

### LINKED ISSUE #9592: Bug: Patient Registration & Doctor Notes is not supporting multi-lines
**Describe the bug**

- [ ] The patient registration form is not allowing users to enter multiple lines in the current address field.
- [ ] The doctor notes field is not reflecting multiple lines when users type in the doctor notes text area.
- [ ] the cards in patient details page for allow transfer and assign volunteer is broken

**To Reproduce**

1. Go to the patient tab
2. Click on Create patient
3. Enter the address in multiple lines

**Expected behavior**

- [ ] The patient registration form for the current address should allow multiple line entries.
- [ ] The doctor notes field should display the text in multiple lines, as entered.
- [ ] keep the card design consistent for allow transfer and assign volunteer

**Screenshots**

Image
Image

### PR DESCRIPTION
**Pull request by AdityaJ2305**
## Proposed Changes

- Fixes #9592 
- Follow up

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [x] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [x] Request for Peer Reviews

## Release Notes

- **UI Improvements**
  - Added `type="button"` to Breadcrumbs back navigation button for better semantic meaning
  - Updated note display in Patient Note Card to prevent text overflow
  - Replaced `AuthorizedButton` with `ButtonV2` in Patient Home component for consistent button styling

- **Behavior Changes**
  - Removed automatic form submission on "Enter" key press

### CODE REVIEW

File: src/components/Facility/PatientNoteCard.tsx [hunk 1]
```diff
@@ -213,7 +213,7 @@ const PatientNoteCard = ({
                 </div>
               </div>
             ) : (
-              <div className="text-sm text-secondary-700 whitespace-pre-wrap">
+              <div className="text-sm text-secondary-700 break-all">
```
Comment (rithviknishad) [inline src/components/Facility/PatientNoteCard.tsx]: this doesn't respect new lines present in the message content right?

File: src/components/Patient/PatientHome.tsx [hunk 2]
```diff
@@ -523,7 +523,7 @@ export const PatientHome = (props: {
 
                     {NonReadOnlyUsers(authUser.user_type) && (
                       <div>
-                        <AuthorizedButton
+                        <ButtonV2
```
Comment (rithviknishad) [inline src/components/Patient/PatientHome.tsx]: Already solved in DO branch, drop these changes

**Review by rithviknishad — CHANGES_REQUESTED** [review]

File: src/components/Facility/PatientNoteCard.tsx (hunk 1 above)
Comment (AdityaJ2305) [reply src/components/Facility/PatientNoteCard.tsx]: working fine 
Screenshot 2024-12-30 at 12 17 04 PM
Screenshot 2024-12-30 at 12 17 32 PM

Comment (rithviknishad) [reply src/components/Facility/PatientNoteCard.tsx]: 1. i don't know any word in dictionary that's that long. this is not needed
2. test for new lines in the message

Comment (AdityaJ2305) [reply src/components/Facility/PatientNoteCard.tsx]: Previously 
Screenshot 2024-12-30 at 12 19 11 PM
Screenshot 2024-12-30 at 12 18 58 PM

Comment (rithviknishad) [reply src/components/Facility/PatientNoteCard.tsx]: image

**Review by rithviknishad — APPROVED** [review]

**Pull request merged**